pymc/distributions/timeseries.py

```diff
@@ -176,127 +310,6 @@ def logp(self, value):
         return at.sum(innov_like) + at.sum(init_like)
 
 
-class GaussianRandomWalk(distribution.Continuous):
-    r"""Random Walk with Normal innovations
-
-    Note that this is mainly a user-friendly wrapper to enable an easier specification
-    of GRW. You are not restricted to use only Normal innovations but can use any
-    distribution: just use `aesara.tensor.cumsum()` to create the random walk behavior.
-
-    Parameters
-    ----------
-    mu : tensor_like of float, default 0
```

Comment:
these (also the ones below) type descriptions are much better than the ones currently proposed, they use `tensor_like` which is rendered as a link to the right definition on the glossary and the default/optional parameter info.